src/sage/rings/polynomial/laurent_polynomial.pyx

```diff
@@ -1629,10 +1628,10 @@ cdef class LaurentPolynomial_univariate(LaurentPolynomial):
 
         The answer is dependent of the base ring::
 
-            sage: S.<u> = LaurentPolynomialRing(QQbar)
-            sage: (2 + 4*t + 2*t^2).is_square()
+            sage: S.<u> = LaurentPolynomialRing(QQbar)                                  # optional - sage.rings.number_field
+            sage: (2 + 4*t + 2*t^2).is_square()                                         # optional - sage.rings.number_field
             False
-            sage: (2 + 4*u + 2*u^2).is_square()
+            sage: (2 + 4*u + 2*u^2).is_square()                                         # optional - sage.rings.number_field
             True
```

Comment:
Hmm. So perhaps `sage.modules` is a dependency of only multi-variate Laurent polynomial ring.

Reply:
yes, see below